diff --git a/node_modules/@types/react/index.d.ts b/node_modules/@types/react/index.d.ts
index b860864..ff0f4a1 100755
--- a/node_modules/@types/react/index.d.ts
+++ b/node_modules/@types/react/index.d.ts
@@ -86,7 +86,7 @@ declare namespace React {
         | (new (props: P) => Component<any, any>);
 
     interface RefObject<T> {
-        readonly current: T | null;
+        readonly value: T | null;
     }
     type RefCallback<T> = { bivarianceHack(instance: T | null): void }["bivarianceHack"];
     type Ref<T> = RefCallback<T> | RefObject<T> | null;
@@ -143,11 +143,21 @@ declare namespace React {
         ref?: LegacyRef<T> | undefined;
     }
 
-    interface ReactElement<P = any, T extends string | JSXElementConstructor<any> = string | JSXElementConstructor<any>> {
+    // interface ReactElement<P = any, T extends string | JSXElementConstructor<any> = string | JSXElementConstructor<any>> {
+    //     type: T;
+    //     props: P;
+    //     key: Key | null;
+    // }
+
+    type ReactElement<P = any, T extends string | JSXElementConstructor<any> = string | JSXElementConstructor<any>> = {
         type: T;
         props: P;
         key: Key | null;
-    }
+    } | (() => {
+        type: T;
+        props: P;
+        key: Key | null;
+    })
 
     interface ReactComponentElement<
         T extends keyof JSX.IntrinsicElements | JSXElementConstructor<any>,
